lua/packer/display.lua

```diff
@@ -101,6 +101,10 @@ local keymaps = {
   prompt_revert = {
     rhs = '<cmd>lua require"packer.display".prompt_revert()<cr>',
     action = 'revert an update'
+  },
+  open_log = {
+    rhs = '<cmd>lua require"packer.display".open_log()<cr>',
+    action = 'open packer logs'
```

Comment:
I'd reword this to `open log file` or something, all the commands are in the context of packer so seems redundant for this one command to explicitly mention packer

Reply:
Yeah, makes sense. Thanks for the suggestion!